### ISSUE TITLE
APIView domain

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-tools#8210]
APIview domain is registered in cloudflare account owned by Jonathan Giles. We need to work with Microsoft domain managers team to transfer this domain or register a new domain so it can be managed centrally by Azure SDK team. Cloudflare support has confirmed that they won't be able to transfer this domain to different account.

**Comment by maririos** [comment]
Could you remind me what was the conclusion from this work? Is this something we still need to worry about?

**Comment by praveenkuttappan** [comment]
Yes, We still need to move it from Jonathan's personal cloudflare account to MSFT account. I didn't have any bandwidth to work on it. We need to check with MS domain manager team to help move this or create a cloudflare account using azure-sdk bot and move it under that user instead of personal cloudflare.
Domain name: apiview.dev

**Comment by tjprescott** [comment]
The registrar for our current custom domains is Wild West Domains.
- apiviewstagingtest.com
- apiviewuxtest.com

@praveenkuttappan who should I contact on the Microsoft domain team to work on this?

**Comment by JonathanGiles** [comment]
I don't know who is responsible for these other domains - not me.

**Comment by tjprescott** [comment]
This has renewed urgency, in part because we cannot block access to APIView from the public internet while it is associated with CloudFlare. Staging and UX test are under Azure control and we can apply service tags to limit traffic to CorpNet, but APIView "sees" this as traffic not from CorpNet, but CloudFlare. Adding the IP ranges for CloudFlare renders it accessible from the public internet.

**Comment by tjprescott** [comment]
We're secured `apiview.org` under App Service Domains and can switch, but we are waiting on a response from the Corporate Domains team to see about transferring apiview.dev first. Marking this blocked until we hear from them.

**Comment by tjprescott** [comment]
@JonathanGiles we heard back from Corporate Domains and this is the transfer process:
https://docs.msftdomains.com/domains/transfer.html

I think as the current owner of the domain, you need to be the Requester because only you can do the necessary steps with CloudFlare.

**Comment by tjprescott** [comment]
This is... DONE.

🥳
